src/foam/u2/view/TableView.js

```diff
@@ -214,7 +306,10 @@ foam.CLASS({
                             column.f ? column.f(obj) : null, obj, column
                           ]).
                           end();
-                      });
+                      }).
+                      add(this.slot(function(editColumnsEnabled) { 
```

Comment:
Does this need to be a slot? Are we going to change this setting on an already created table?